dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java

```diff
@@ -0,0 +1,188 @@
+
+        //File split
+        FileSplit fileSplit = new FileSplit(datafile);
+
+
+        //set CSV Record Reader and initialize it
+        RecordReader rr= new CSVRecordReader(1,',');
+        rr.initialize(fileSplit);
+
+        //Build Schema to prepare the data
+        Schema sc = new Schema.Builder()
+                .addColumnInteger("id")
+                .addColumnsFloat("huml","humw","ulnal","ulnaw","feml","femw","tibl","tibw","tarl","tarw")
+                .addColumnCategorical("type", Arrays.asList("SW","W","T", "R", "P", "SO"))
+                .build();
+
+        TransformProcess tp = new TransformProcess.Builder(sc)
+                .removeColumns("id")
+                .categoricalToInteger("type")
+                .build();
+
+//        Checking the schema
+        Schema outputschema=tp.getFinalSchema();
+        System.out.println(outputschema);
+
+
+        List<List<Writable>> alldata = new ArrayList<>();
+
+        while(rr.hasNext()){
+            alldata.add(rr.next());
+        }
+
+        List<List<Writable>> processdata =LocalTransformExecutor.execute(alldata, tp);
+
+        //Create iterator from process data
+        CollectionRecordReader collectionRR = new CollectionRecordReader(processdata);
+
+        //Input batch size , label index , and number of label
+        DataSetIterator dataSetIterator = new RecordReaderDataSetIterator(collectionRR, processdata.size(),10,6);
+
+//        //Create Iterator and shuffle the data
+        //Error due to zero data is obtain
+        DataSet fulldataset = dataSetIterator.next();
+        fulldataset.shuffle();
+//
+//        //Input split ratio
+        SplitTestAndTrain testAndTrain = fulldataset.splitTestAndTrain(splitratio);
+//
+          //Get train and test dataset
+        DataSet trainData =testAndTrain.getTrain();
+        DataSet testData =testAndTrain.getTest();
+
+        //printout size
+        System.out.println("Training vector : ");
+        System.out.println(Arrays.toString(trainData.getFeatures().shape()));
+        System.out.println("Test vector : ");
+        System.out.println(Arrays.toString(testData.getFeatures().shape()));
+
+        //Data normalization
+        DataNormalization normalizer = new NormalizerMinMaxScaler();
+        normalizer.fit(trainData);
+        normalizer.transform(trainData);
+        normalizer.transform(testData);
+
+
+        //Get network configuration
+        MultiLayerConfiguration config = getConfig(numInput, numClass, learningrate);
+
+
+
+        //Define network
+        MultiLayerNetwork model = new MultiLayerNetwork(config);
+        model.setListeners(new ScoreIterationListener(10));
+        model.init();
+
+        //Evaluator
+        UIServer server = UIServer.getInstance();
+        StatsStorage storage = new InMemoryStatsStorage();
+        server.attach(storage);
+
+        //Training
+        for (int i = 0; i < epoch; ++i) {
```

Comment:
this for loop can reduced to a one-liner `model.fit(trainData, epoch)` if you are not doing anything else in the for loop